dist/styles/angular-patternfly.css

```diff
@@ -171,3 +171,95 @@
 .data-list-pf .row-column {
   padding-right: 5px;
 }
+
+.tiles-view-pf {
+  overflow: auto;
+  padding-top: 20px;
+}
+
+.tiles-view-pf .tile {
+  -webkit-box-shadow: 0 1px 1px rgba(0, 0, 0, .175);
+  background: #fff;
+  border: 1px solid #eee;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.175);
+  display: block;
+  float: left;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  padding: 10px;
+  position: relative;
+  text-align: center;
+}
+
+.tiles-view-pf .tile-check-box {
+  left: 10px;
+  position: absolute;
+  top: 8px;
+  width: 20px;
+  z-index: 3;
+}
+
+.tiles-view-pf .tile-content {
+  height: 100%;
+  left: 0px;
+  margin: 10px;
+  padding-right: 40px;
+  position: absolute;
+  top: 0px;
```

Comment:
I think the issue is with your top margin. You probably don't want to use position absolute here. If you remove all the styles and just have .tiles-view-pf .tile-content { margin: 2px 40px 10px 10px; overflow: auto; } it should get you what you want.